source/boundary_composition/interface.cc

```diff
@@ -315,6 +326,16 @@ namespace aspect
                            "implemented in a plugin in the BoundaryComposition "
                            "group, unless an existing implementation in this group "
                            "already provides what you want.");
+        prm.declare_entry ("Allow fixed composition on outflow boundaries", "false",
+                           Patterns::Bool (),
+                           "When the composition is fixed on a given boundary as determined "
+                           "by the list of 'Fixed composition boundary indicators', there "
+                           "might be parts of the boundary where material flows out and "
+                           "one may want to prescribe the composition only on the parts of "
+                           "the boundary where there is inflow. This parameter determines "
```

Comment:
the -> those

Reply:
I don't understand this comment (but fixed the one above for line 334). Was this what you meant?

Reply:
It's just a question of emphasis. "only on the parts the boundary" and "only on those parts of the boundary" means the same, it just provides additional emphasis.